buildSrc/src/main/groovy/io/micronaut/guides/GuideAsciidocGenerator.groovy

```diff
@@ -133,23 +138,63 @@ class GuideAsciidocGenerator {
         List<String> rawLines = []
 
         for (String rawLine : lines) {
-            if (rawLine.startsWith('include::{commondir}/') && rawLine.endsWith('[]')) {
-                String commonFileName = rawLine.substring(rawLine.indexOf('include::{commondir}/') + 'include::{commondir}/'.length(), rawLine.indexOf('[]'))
-
-                File commonFile = Paths.get(destinationFolder.absolutePath, "../common/$commonFileName").toFile()
-                assert commonFile.exists()
-
-                rawLines.add("// Start: ${commonFileName}".toString())
-                rawLines.addAll(expandAllCommonIncludes(commonFile.readLines(), destinationFolder))
-                rawLines.add("// End: ${commonFileName}".toString())
```

Comment:
I think we should keep the `// Start` and `// End` comments because they help debugging the guide generated. It's not something that we use everyday but when there is a problem with our macros it helps to see from which snippet a specific text is coming.

Reply:
I think, at most, this should be a debug setting. Not a default setting.